exo1/lib/screens/users/users_screen.dart

```diff
@@ -1,15 +1,27 @@
+import 'package:exo1/domain/user.dart';
 import 'package:exo1/domain/user_repository.dart';
 import 'package:exo1/screens/users/user_list_widget.dart';
 import 'package:flutter/material.dart';
 
 import 'add_user_toolbar.dart';
 
-class UsersScreen extends StatelessWidget {
+class UsersScreen extends StatefulWidget {
```

Comment:
There is a Flutter widget that would allow you to update the screen with `repository.observeUsers()` without using a `StatefulWidget` 😉